migrate/migrate_repos.py

```diff
@@ -0,0 +1,74 @@
+import itertools
+import os
+import sys
+import time
+
+import click
+from ghapi.all import GhApi, paged
+
+@click.command()
+@click.argument('src_org')
+@click.argument('dest_org')
+@click.argument('repo_list_file', type=click.File('r'))
+@click.option(
+    '--preview',
+    is_flag=True,
+    help="Preview what will happen but don't execute."
+)
+@click.option(
+    '--skip-missing',
+    is_flag=True,
+    help="Skip repos that are not found in the source org, instead of failing. Useful after an error."
+)
+def migrate(src_org, dest_org, repo_list_file, preview, skip_missing):
+    if preview:
+        click.echo("In Preview Mode: No changes will be made!")
+
+    if src_org == dest_org:
+        sys.exit("Fatal Error: Source and destination orgs must be different.")
+
+    github_token = os.environ.get('GITHUB_TOKEN')
+    if not github_token:
+        sys.exit("Fatal Error: Please set a GITHUB_TOKEN environment variable.")
+
+    # Get the list of repos from our text file.
+    repos_to_transfer = [
+        repo_name.strip() for repo_name in repo_list_file if repo_name.strip()
```

Comment:
I imagine it would be helpful to be able to comment out lines in this text file.

Reply:
Ah, good point. That's easy enough to add.